libs/extract/src/lib/helperFunction.ts

```diff
@@ -1,3 +1,5 @@
+import * as React from 'react'
+
 export type IndicatorType = 'valid' | 'invalid' | 'neutral'
```

Comment:
maybe we can replace neutral with disabled as if the form is neutral, means it is valid

Reply:
The neutral actual displays error message without any specific color or border, so I renamed to it something better ?

Reply:
`info`?

Reply:
Yeap sounds good to me